config.yml.default

```diff
@@ -1,131 +1,131 @@
-token: "xxxxxxxxxxxxxxxx"    # Lichess OAuth2 Token.
-url: "https://lichess.org/"  # Lichess base URL.
+token: "lip_yourtokenherexxxxx"    # Lichess OAuth2 Token.
```

Comment:
I like the extra spaces that you add in all the lines. i think increasing the number of `x`s to 24 is better than `lip_yourtokenherexxxxx`.

Reply:
Changed